src/status_im/data_store/realm/accounts.cljs

```diff
@@ -1,20 +1,25 @@
 (ns status-im.data-store.realm.accounts
   (:require [status-im.data-store.realm.core :as realm]))
 
+(defn- reformat-networks [account]
+  (update account :networks
+          (fn [networks]
+            (into {}
+                  (map (fn [{:keys [id] :as network}]
+                         [id network])
+                       (vals (js->clj networks :keywordize-keys true)))))))
+
 (defn get-all []
   (realm/get-all realm/base-realm :account))
 
 (defn get-all-as-list []
-  (realm/realm-collection->list (get-all)))
+  (map reformat-networks (realm/realm-collection->list (get-all))))
 
 (defn get-by-address [address]
-  (realm/get-one-by-field-clj realm/base-realm :account :address address))
+  (reformat-networks
+    (realm/get-one-by-field-clj realm/base-realm :account :address address)))
 
 (defn save [account update?]
   (realm/write realm/base-realm
-    #(realm/create realm/base-realm :account account update?)))
-
-(defn save-all [accounts update?]
-  (realm/write realm/base-realm
-               (fn []
-                 (mapv #(realm/create realm/base-realm :account % update?) accounts))))
+    (let [account' (update account :networks (comp vec vals))]
```

Comment:
Can we call this something other than `account'`? Why do we get rid of network keys (`vals`) when persisting?

Reply:
> Can we call this something other than account'?
why?

> Why do we get rid of network keys (vals) when persisting?
realm doesn't support this

Reply:
Wrote in Slack but putting it here too. All these `account'` etc comments: idea is just that adding `'` is not very descriptive of what it is or how it is different, if possible it's nice to capture this in variable name imo

Reply:
code after that name can be descriptive enough. Again, imo, adding something
```clojure
(let [this-name-means-that-something-was-done-with-account (doing something with account)]
...)
```
to these intermediate symbols doesn't worth efforts. They are useful because can make code which uses them later more clear (instead of inlining some additional stuff), but eh, adding `'` is enough imo. We can move this discussion to slack, sure, i haven't seen it